media/css/custom.scss

```diff
@@ -521,3 +526,32 @@ summary {
     color: $white;
   }
 }
+
+.dictionary-container a {
+  color: $white;
+  float: left;
+  font-size: 50px;
+  line-height: 80px;
+  list-style: none;
+  margin-left: 37px;
+  margin-top: 25px;
+  text-align: center;
+  text-decoration: none;
+  width: 80px;
+}
+
+.dictionary-container a:nth-child(8n+1) {
```

Comment:
Why not move `a:nth-child` under the previous `.dictionary-container` definition?